ffmap/routertools.py

```diff
@@ -246,10 +245,9 @@ def parse_nodewatcher_xml(xml):
 				"clients": int(tree.xpath("/data/client_count/text()")[0]),
 				"has_wan_uplink": (
 					(len(tree.xpath("/data/system_data/vpn_active")) > 0
-					and int(tree.xpath("/data/system_data/vpn_active/text()")) == 1)
+					and int(tree.xpath("/data/system_data/vpn_active/text()")[0]) == 1)
 					or len(tree.xpath("/data/interface_data/%s" % CONFIG["vpn_netif"])) > 0
-					or len(tree.xpath("/data/interface_data/%s" % CONFIG["vpn_netif_l2tp"])) > 0
-					or len(tree.xpath("/data/interface_data/%s" % CONFIG["vpn_netif_l2tp1"])) > 0
+					or len(tree.xpath("/data/interface_data/%s0" % CONFIG["vpn_netif_l2tp"])) > 0
```

Comment:
This is just an intermediate solution matching l2tp0. Please suggest a solution with wildcard/regex using xpath, as I was unable to do so.